### PR TITLE
I added upload file and upload a new text

### PR DESCRIPTION
**Pull request by BilginK**
I hope it was a problem-free pull request.

### CODE REVIEW

**Review by ibrhmucar — APPROVED** [review]

File: configuration.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-browser= remote_chrome
```
Comment (Umit-Sahin) [inline configuration.properties:1]: Origin should remain "remote_crome"

File: src/test/resources/features/Files.feature [hunk 2]
```diff
@@ -17,12 +18,38 @@ Feature: Add - Delete - Move File Functionalities
     And verify new "Şükriye_Doğaç!" file is shown on the page
 
 
-  Scenario: User can upload a file
+	#{color:#de350b}*US*{color}
```
Comment (Umit-Sahin) [inline src/test/resources/features/Files.feature:21]: Line 21 and 25 could be delete

File: src/test/resources/features/Files.feature [hunk 3]
```diff
@@ -17,12 +18,38 @@ Feature: Add - Delete - Move File Functionalities
...
+	#
+	# A word file welcome
+  @ETSKY-394
+  Scenario: Upload a new file
     Given the user logged in
     When  user should be click the add icon module
     Then user should be able to upload "HelloMeetSky" file name
     And verify new "HelloMeetSky" file is shown on the page
 
+#{color:#de350b}*US*{color}
```
Comment (Umit-Sahin) [inline src/test/resources/features/Files.feature:35]: Line 35 and 39 could be deleted

File: target/test-classes/features/Files.feature [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
 Feature: Add - Delete - Move File Functionalities
 
 	#{color:#ff0000}*US*{color}
```
Comment (Umit-Sahin) [inline target/test-classes/features/Files.feature:4]: Line 4,21,25 could be delete

File: src/test/java/net/meetsky/runners/SkyRunner.java [hunk 5]
```diff
@@ -16,7 +16,7 @@
         strict = true,
         dryRun =false,
 
-        tags = "@ETSKY-325"
+        tags = " @ETSKY-395"
```
Comment (Umit-Sahin) [inline src/test/java/net/meetsky/runners/SkyRunner.java:19]: In generally tags should remain @smoke in order to get smoke test result

**Review by Umit-Sahin — CHANGES_REQUESTED** [review]

**Pull request merged**